### PR TITLE
Fix/colors windows

### PR DESCRIPTION
**Pull request by victor-schumacher**
**- What I did**
Changed the colors.go, now using the gookit/color lib.

**- How to verify it**
Use commands and check colors

**- Description for the changelog**
Changed the colors.go, now using the gookit/color lib.

### CODE REVIEW

File: pkg/cmd/set_credential.go [hunk 1]
```diff
@@ -218,7 +218,7 @@ func (s setCredentialCmd) stdinResolver() (credential.Detail, error) {
 		return credDetail, nil
 	}
 
-	return credDetail, fmt.Errorf(prompt.Red, "invalid CLI build, no edition defined")
+	return credDetail, prompt.NewError( "invalid CLI build, no edition defined")
```
Comment (kaduartur) [inline pkg/cmd/set_credential.go:221]: Remove space

File: pkg/formula/creator.go [hunk 2]
```diff
@@ -26,11 +25,11 @@ const (
 )
 
 var (
-	ErrDontStartWithRit = fmt.Errorf(prompt.Red, "Rit formula's command needs to start with \"rit\" [ex.: rit group verb <noun>]")
-	ErrTooShortCommand  = fmt.Errorf(prompt.Red, "Rit formula's command needs at least 2 words following \"rit\" [ex.: rit group verb]")
-	ErrRepeatedCommand  = fmt.Errorf(prompt.Red, "this command already exists")
-	ErrTreeJsonNotFound = fmt.Errorf(prompt.Red, "tree.json not found")
-	ErrMakefileNotFound = fmt.Errorf(prompt.Red, "makefile not found")
+	ErrDontStartWithRit = prompt.NewError( "Rit formula's command needs to start with \"rit\" [ex.: rit group verb <noun>]")
```
Comment (kaduartur) [inline pkg/formula/creator.go:28]: Remove spaces

File: pkg/prompt/color.go [hunk 3]
```diff
@@ -1,26 +1,45 @@
...
-	Teal   = "\033[1;36m%s\033[0m"
-	Yellow = "\033[1;33m%s\033[0m"
-	Red    = "\033[1;31m%s\033[0m"
-	Green  = "\033[1;32m%s\033[0m"
+	"github.com/gookit/color"
 )
 
-func Error(text string) {
-	fmt.Println(fmt.Sprintf(Red, text))
+// Returns new error with red message
```
Comment (kaduartur) [inline pkg/prompt/color.go:10]: The good practice in Golang is starting a comment with the function name.
e.g.: NewError returns a new error with a red message

Comment (victor-schumacher) [reply pkg/prompt/color.go]: Sure! Thanks for reviewing :100:

**Pull request merged**